lib/ansible/plugins/callback/profile_tasks.py

```diff
@@ -42,6 +42,16 @@
         ini:
           - section: callback_profile_tasks
             key: sort_order
+      summary_only:
+        description: Only show summary, not individual task profiles
+        type: bool
+        default: False
+        env:
+          - name: PROFILE_TASKS_SUMMARY_ONLY
+        ini:
+          - section: callback_profile_tasks
+            key: summary_only
+            version_added: '2.7'
```

Comment:
This needs to be '2.8' since freeze dates for 2.7 have passed.

Reply:
also incorrectly indented, it applies to the whole option, not just the configuration entry